### ISSUE TITLE
Do not delete files and folders at installing

### ISSUE THREAD

**Issue by imaginaryusername** [Electron-Cash/Electron-Cash#1259]
At least one user reported that installing the installable version in a portable folder seems to remove wallet files. While not a normally expected behavior, in general it seems desirable for an installer to not remove any files unnecessarily, and only overwrite them where needed.

**Comment by tulasacra** [comment]
steps to reproduce:
1. install the portable version to directory \EC
2. run, create a default wallet (\EC\electron_cash_data\wallets\default_wallet)
3. install Electron-Cash-4.0.1-setup.exe to directory \EC
4. poof! your wallet is gone :)

![image](https://user-images.githubusercontent.com/5469363/55583409-17b8c380-5722-11e9-99f4-26059176c7cb.png)

**Comment by cculianu** [comment]
Jesus christ.  Hell.. that is NOT desirable behavior.  I'll look into it.  Probably need to tweak the electron-cash.nsi file we use to build the installer.  Thanks for reporting

**Comment by cculianu** [comment]
Ok well what you did is a bit non-standard .. but I made the installer warn you if it's blowing away a directory with existing files.

It's not easy to make the installer detect the EC directory inside from the installer scripting language -- what you did is really non-standard.. but if it happened to you it can happen to others.

Best I can offer is the installer now has an All-stop dialog asking you to confirm that you want to delete existing files if it detects existing files.

Fixed in latest commit.
